api/envoy/api/v2/listener/listener.proto

```diff
@@ -215,3 +215,22 @@ message ListenerFilter {
     google.protobuf.Any typed_config = 3;
   }
 }
+
+// [#proto-status: experimental]
+message QuicListener {
+  // The name of the QUIC listener to instantiate. The name must match a
+  // supported listener. Future listeners may include:
+  //
+  // * envoy.quic.quiche
+  // * envoy.quic.ngtcp2
+  string name = 1 [(validate.rules).string.min_bytes = 1];
+
+  // Filter specific configuration which depends on the filter being instantiated.
+  // See the supported filters for further documentation.
+  oneof config_type {
+    google.protobuf.Struct config = 2;
+
+    // [#not-implemented-hide:]
```

Comment:
cc @lizan for whichever PR merges first.

Reply:
Yeah, I think we can go straight to `Any` and skip the `Struct` in this PR.